platform/ios/PolyPodApp/PolyPod/SettingsView.swift

```diff
@@ -160,6 +188,35 @@ private struct SettingsButton: View {
     }
 }
 
+typealias OnChange = ((Bool) -> Void)?
+
+private struct SettingsToggleButton: View {
+    let label: LocalizedStringKey
+    let isToggle : Binding<Bool>;
+        
+    var onChange: OnChange
+
+    var body: some View {
+       VStack {
+           Toggle(isOn: isToggle.onChange(toggleChange)) {
+                    Text(label)
+                           .foregroundColor(Color.PolyPod.darkForeground)
+                           .font(.custom("Jost-Regular", size: 18))
+                           .kerning(-0.18)
```

Comment:
This is taken from Figma, right?

Reply:
it's the same as the other labels on the settings. there's no figma for the tuggle button so far. I guess it's ok for now as it looks like the others ?!

Reply:
Well, you've got all kind of kerning settings, some possibly indistinguishable:

```
platform/ios/PolyPodApp/PolyPod/FeatureListView.swift:                    .kerning(-0.2)
platform/ios/PolyPodApp/PolyPod/FeatureListView.swift:                    .kerning(-0.14)
platform/ios/PolyPodApp/PolyPod/FeatureListView.swift:                    kerning: -0.16,
platform/ios/PolyPodApp/PolyPod/FeatureView.swift:            .kerning(-0.16)
platform/ios/PolyPodApp/PolyPod/OnboardingView.swift:                kerning: -0.38,
platform/ios/PolyPodApp/PolyPod/OnboardingView.swift:                kerning: -0.38,
platform/ios/PolyPodApp/PolyPod/OnboardingView.swift:                kerning: -0.24,
platform/ios/PolyPodApp/PolyPod/OnboardingView.swift:                        .kerning(-0.18)
platform/ios/PolyPodApp/PolyPod/ParagraphView.swift:    private let kerning: CGFloat?
platform/ios/PolyPodApp/PolyPod/ParagraphView.swift:        kerning: CGFloat? = nil,
platform/ios/PolyPodApp/PolyPod/ParagraphView.swift:        self.kerning = kerning
platform/ios/PolyPodApp/PolyPod/ParagraphView.swift:        kerning: CGFloat? = nil,
platform/ios/PolyPodApp/PolyPod/ParagraphView.swift:        self.kerning = kerning
platform/ios/PolyPodApp/PolyPod/ParagraphView.swift:                kerning: kerning,
platform/ios/PolyPodApp/PolyPod/ParagraphView.swift:    var kerning: CGFloat? = nil
platform/ios/PolyPodApp/PolyPod/ParagraphView.swift:        if let kerning = kerning {
platform/ios/PolyPodApp/PolyPod/ParagraphView.swift:            attributes[NSAttributedString.Key.kern] = kerning
platform/ios/PolyPodApp/PolyPod/SettingsView.swift:                    .kerning(-0.16)
platform/ios/PolyPodApp/PolyPod/SettingsView.swift:            .kerning(-0.12)
platform/ios/PolyPodApp/PolyPod/SettingsView.swift:                .kerning(-0.18)
```

That's why I wanted to make sure you were taking it from the actual design. At any rate, all these literal numbers make me a bit nervous, and defining a constant for it or including in resources or whatever is done in this platform might not be such a bad idea.

Reply:
are these values coming from design always? I agree we should have a more generic way and not decimals (that not make any sense :S). maybe future-work ?

Reply:
I don't see other alternative... It's always design, or feedback from them.